CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-## [0.14.0](https://github.com/eclipse/lemminx/milestone/18?closed=1) (November 6, 2020)
+## [0.14.0](https://github.com/eclipse/lemminx/milestone/18?closed=1) (November 10, 2020)
```

Comment:
0.14.0 is released, that's 0.14.1 now, with 2 bug fixes

Reply:
fixed